content/blogs/investigating-wddm.md

```diff
@@ -0,0 +1,40 @@
+---
+title:      "An Initial investigation into WDDM on ReactOS"
+author:     "The_DarkFire_"
+date:       2025-10-3
+tags:       [ newsletter, wddm ]
+---
+
+ReactOS’s history spans a wider range than the lives of many of the people who work on it today. Incredible individuals have come and gone from the project with vastly different goals for what they want to see developed. In recent years It’s become more of an interest to work on hardware support. As ReactOS brings its gaze onto the world of Vista a few topics regarding how hardware works make themselves known. Vista introduced massive overhauls to how hardware drivers are written and maintained; Gradually we’re trying to handle many of these overhauls with some great success. The blog post today is about WDDM, or the Windows Display Driver Model.
+
+### The “Overhaul”
+I’ve always found the video driver documentation lacking for both architectures, enough to get started but not really enough to get deep into the world of 3D. Thankfully in recent years this has changed enough to where some open source, individual written drivers have popped up. In essence WDDM is an overhaul that splits responsibility of the GPU away from win32k and provides a far more fluid, and evolving control over the GPU to the driver vendor. Dxgkrnl.sys talks to a miniport driver to have different levels of the interface filled in. The WDDM revision (WDDM 1.0, 1.1, 1.2….) only really describes how much of these interfaces are supported and implemented. This is different than the feature level that is described in DxDiag. 
+(note: maybe a picture of win10 dxdiag here).
+
+### Wait… What happened to XDDM?
+	Officially starting with Windows 8 every GPU driver for the system had to be a WDDM driver. But all that was really dropped was the miniport portion. Vista/7 could always still load XP gpu drivers without a complaint. 
```

Comment:
Sometimes there is this indenting, and sometimes not.
I think it's better to choose one style.